ui/src/components/harvest-games/HarvestGameInstructions.js

```diff
@@ -0,0 +1,357 @@
+              borderRadius="10px"
+              fontWeight={500}
+              fontSize={{ base: '5vw', md: '2.5vw' }}
+              textColor="rgba(61, 61, 61, 1)"
+              w="100%"
+              marginX="0"
+              marginY="5"
+              textAlign="start"
+              paddingY="5"
+              paddingX="9"
+            >
+              Materials :
+              <VStack justifyContent="space-evenly" alignItems="start">
+                {gameType[selected].id === 'game_2' &&
+                  gameType[selected].materials.length > 0 &&
+                  gameType[selected].materials.map((e, i) => {
+                    return (
+                      <Fragment key={i}>
+                        <Text
+                          color="rgba(1, 43, 85, 1)"
+                          fontWeight={400}
+                          textDecoration="none"
+                          fontSize={{ base: '4vw', md: '2vw' }}
+                        >
+                          {e}
+                        </Text>
+                      </Fragment>
+                    );
+                  })}
+              </VStack>
+            </Box>
+            <Heading paddingX="2">Instructions :</Heading>
+            <Grid
+              w="100%"
+              mt="12"
+              mb="12"
+              templateColumns="repeat(1, 1fr)"
+              gap="2"
+            >
+              {gameType[selected].imageUrl.length > 0 &&
+                gameType[selected].imageUrl.map((url, i) => (
+                  <Box
+                    minH={{ base: '60vh', md: '80vh' }}
+                    w="full"
+                    fontWeight="700"
+                    fontSize={{ base: '60', md: '70' }}
+                    textColor="rgba(1, 43, 85, 1)"
+                    bgImage={url}
+                    bgSize="contain"
+                    bgRepeat="no-repeat"
+                    bgPos="center"
+                    marginX="auto"
+                    textAlign="end"
+                    paddingBottom="8"
+                  >
+                    {i + 1}
+                  </Box>
+                ))}
+            </Grid>
+          </VStack>
+        </Box>
+      ) : (
+        <VStack
+          display="flex"
+          w="100%"
+          fontSize={{ md: '16', lg: '18' }}
+          textColor="rgba(83, 83, 83, 1)"
+          alignItems={{ base: 'center', sm: 'start' }}
+          textAlign={{ base: 'center', sm: 'start' }}
+          paddingX="5"
+          spacing="4"
+        >
+          <label>
+            Please enter the password for {gameType[selected].game_num}{' '}
+            instructions
+          </label>
+          <Input
+            id="inputted"
+            type="text"
+            placeholder="Start typing here"
+            w={{ base: '100%', sm: '50%' }}
+            backgroundColor="white"
+            borderWidth="1px"
+            borderColor="gray/300"
+            onKeyDown={(e) => handleKeyDown(e)}
```

Comment:
side note, if you make this a form u don't need to deal with this but good workaround haha